lib/shopify-cli/app_types/rails.rb

```diff
@@ -31,7 +31,7 @@ def open_url
         end
 
         def webhook_location
-          "config/webhooks"
+          "shopify_app.rb"
```

Comment:
What is this doing? Is this the initializer? Should it be config/initializers/shopify_app.rb?

Reply:
Yes, I'll put the full path. This is just where shopify_app generates the code.